libvmi/driver/kvm/kvm.c

```diff
@@ -1436,6 +1623,15 @@ kvm_init_vmi(
     dbprint(VMI_DEBUG_KVM, "--SUCCESS\n");
 #endif
 
+#ifdef HAVE_LIBKVMI
+    dbprint(VMI_DEBUG_KVM, "--Connecting to KVMI...\n");
+    if (!init_kvmi(kvm)) {
+        dbprint(VMI_DEBUG_KVM, "--KVMI failed\n");
+        return VMI_FAILURE;
+    }
+    dbprint(VMI_DEBUG_KVM, "--KVMI connected \o/\n");
```

Comment:
I understand your excitation, i share it, but i think you can remove the emoji \o/ here ;)